server/tools/video_tools.py

```
@@ -0,0 +1,252 @@
+    # Get video model configuration
+    model_info = ctx.get('model_info', {})
+    video_model = model_info.get('video')
+    
+    # Always use default Fal AI Hailuo model if no video model is configured
+    if not video_model or video_model is None:
+        video_model = {
+            'model': 'minimax/hailuo-02',
+            'provider': 'fal'
+        }
+    
+    model = video_model.get('model', 'minimax/hailuo-02')
```

Flake8 (7.2.0)

[error] local variable 'model' is assigned to but never used

(F841)